tokens/governance/treasury-spend.md

```diff
@@ -1,97 +1,43 @@
-    | Treasury Address | [0x6d6f646C70792f74727372790000000000000000](https://moonriver.subscan.io/account/0x6d6f646C70792f74727372790000000000000000){target=\_blank} |
-
-=== "Moonbase Alpha"
-    |     Variable     |                                                                    Value                                                                    |
-    |:----------------:|:-------------------------------------------------------------------------------------------------------------------------------------------:|
-    |  Proposal Bond   |                                      {{ networks.moonbase.treasury.proposal_bond}}% of proposed spend                                       |
-    |   Minimum Bond   |                                            {{ networks.moonbase.treasury.proposal_bond_min}} DEV                                            |
-    |   Spend Period   |                                           {{ networks.moonbase.treasury.spend_period_days}} days                                            |
-    | Treasury Address | [0x6d6f646C70792f74727372790000000000000000](https://moonbase.subscan.io/account/0x6d6F646c70632f74727372790000000000000000){target=\_blank} |
-
-This guide will show you how to submit a proposal on Moonbase Alpha. It can be adapted for Moonbeam or Moonriver. In any case, it's recommended that you familiarize yourself with the steps of submitting a Treasury proposal on Moonbase Alpha or a local dev node before taking the steps on Moonbeam or Moonriver.
-
-## Roadmap of a Treasury Proposal {: #roadmap-of-a-treasury-proposal }
+ - **Value** — the amount that is being asked for and will be allocated to the beneficiary address if the Treasury proposal is passed
 
-You can find a full explanation of the [happy path for a Treasury proposal on the Treasury overview page](/learn/features/treasury/){target=\_blank}. For more information on the process, see this [guide to the interim community Treasury program](https://moonbeam.network/news/proposal-for-interim-treasury-program-approved-by-moonbeam-community){target=\_blank}.
-
-![Proposal Roadmap](/images/tokens/governance/treasury-proposals/treasury-proposal-roadmap.webp)
-
-## Submitting your Idea to the Forum {: #submitting-your-idea-to-the-forum }
-
-It's highly recommended that you preface any proposal with a post on [Moonbeam's Community Forum](https://forum.moonbeam.network){target=\_blank}. You should allow a period of five days for the community to provide feedback on the Moonbeam Forum post. This is especially important for Treasury proposals because of the irrevocable bond required to submit a Treasury spend proposal. You can submit your idea for a Treasury spend proposal by following [the instructions in the Using the Moonbeam Community Forum to Submit a Treasury Proposal guide](https://moonbeam.network/news/using-the-moonbeam-community-forum-to-submit-a-treasury-proposal){target=\_blank}.
+--8<-- 'text/learn/features/treasury/treasury-addresses-path.md'
```

Comment:
Again, I don't think it makes sense to have this under definitions. It made more sense when we had a table that corresponded to each of the definitions. But now definitions feels very specific to terminology

Maybe we need to rename the section to Parameters Required to Submit a Proposal or something - but even then, the Treasury address isn't required. So I don't know. I just don't think Definitions makes sense anymore